src/main/java/com/loievroman/bookstoreapp/repository/UserRepository.java

```diff
@@ -3,9 +3,11 @@
 import com.loievroman.bookstoreapp.model.User;
 import java.util.Optional;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface UserRepository extends JpaRepository<User, Long> {
+    @Query("SELECT u FROM User u JOIN FETCH u.roles WHERE u.email = :email")
```

Comment:
Or you can use more readable annotation
```suggestion
    @EntityGraph(attributePaths = "roles")
```